internal/srv/handlers.go

```diff
@@ -29,6 +31,8 @@ func (s *Server) processEvent(messages <-chan *message.Message) {
 		if err != nil {
 			s.Logger.Errorw("unable to unmarshal event message", "error", err)
 			msg.Nack()
+
+			continue
```

Comment:
Why are we continuing here ? The `nack` should end the processing right?

Reply:
the nack didn't break the loop processing fwict